core/src/main/scala/org/apache/spark/ui/PagedTable.scala

```diff
@@ -123,10 +123,9 @@ private[ui] trait PagedTable[T] {
   /**
    * Return a page navigation.
    * <ul>
-   *   <li>If the totalPages is 1, the page navigation will be empty</li>
    *   <li>
-   *     If the totalPages is more than 1, it will create a page navigation including a group of
-   *     page numbers and a form to submit the page number.
+   *     It will create a page navigation including a group of page numbers and a form
```

Comment:
I don't think we need to put this comment inside `<ul><li>..</li></ul>`

Reply:
true.

Reply:
@gengliangwang @felixcheung 
i have updated according to your suggestion. please check.